portfolio/src/main/webapp/script.js

```diff
@@ -47,4 +47,18 @@ function randomPic() {
   picElem.src = pics[randomFromArray(pics)]
   //make image visible
   picElem.style.display = "block"
+}
+
+/**
+ * Do hello is a dynamic approach to the /hello servlet
```

Comment:
For commenting functions: 3rd person is preferred. So something like
"Fetches the response dynamically from the /hello servelet." would be better